front/app/translations/es-CL.json

```diff
@@ -1096,6 +1096,7 @@
   "app.containers.Projects.projectTwitterMessage": "¡Haz que se escuche tu voz! Participa en {projectName} | {orgName}",
   "app.containers.Projects.projects": "Proyectos",
   "app.containers.Projects.questions": "Preguntas",
+  "app.containers.Projects.readLess": "Read less",
```

Comment:
This "Read less" seems to have not been translated for many important languages. These translations were not in the Crowdin `master` [PR](https://github.com/CitizenLabDotCo/citizenlab/pull/2442) I merged into `master`.

Should I continue with this release? (and we somehow get the translations corrected). Or should I follow some other course of action?

@sebastienhoorens @IvaKop Either of you have an idea?

Reply:
@alexander-cit These translations were in the Crowdin `master` [PR](https://github.com/CitizenLabDotCo/citizenlab/pull/2432/files) you merged 5 days ago. Do you know anything about the untranslated `"app.containers.Projects.readLess": "Read less"` entries?

Reply:
As far as I know, the presence of this copy in the `es-CL.json` file doesn't change anything. If it weren't here, the English version would be used anyway. The only thing that matters is the code that uses it. My and this PR doesn't include it, so we shouldn't care about it. I mean, we could revert that change, but I don't think it's that critical.

And in general, I think we can just trust Crowdin managers/translators and merge more or less anything we see if it looks normal.

Reply:
OK, thanks @alexander-cit.
I guess my thinking is that if we accept it, then the translators won't be notified that it needs translating. But I agree, it won't break code.
I'll proceed with the release, and check with @brentguf when he's back to make sure we follow up if needed to ensure these strings are translated.

Reply:
I suppose it's normal Crowdin flow when first it writes an English copy to all the files, and then it writes a translated copy.

Reply:
Probably it is, and maybe I once understood that. I seem to find it really hard to hold Crowdin knowledge in my brain.